network/src/main/scala/com/linkedin/norbert/network/netty/DarkCanaryChannelHandler.scala

```diff
@@ -69,13 +70,20 @@ class DarkCanaryChannelHandler extends Logging {
   private var clusterClient : Option[ClusterClient] = None
   private var staleRequestTimeoutMins : Int = 0
   private var staleRequestCleanupFrequencyMins : Int = 0
-  private var upstreamCallback: Option[(Boolean, UUID, Object)=>Unit] = None // Boolean: FromDark, UUID: RequestID, Object: Msg
+  private var upstreamCallback: Option[(Boolean, UUID, Request[Any,Any], NorbertProtos.NorbertMessage)=>Unit] = None // Boolean: FromDark, UUID: RequestID, Object: Msg
```

Comment:
Just to keep it simple, can we use DarkCanaryResponseHandler instead of just tracking the call back method?
private var darkCanaryResponseHandler: Option[DarkCanaryResponseHandler] = None